RegionalTestApp/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.Identity.Client;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+
+/**
+ * 1. On an Azure VM, copy the code in a console app
+ * 2. Run the app and verify the token is obtained.
+ * 3. From the logs verify the token endpoint should contain the region. eg. centralus or eastus
+ **/
+namespace TestApp
+{
+    class Program
+    {
+        static string clientId = "<Lab_PublicCloudConfidentialClientID>";
+#pragma warning disable UseAsyncSuffix // Use Async suffix
+        static async Task Main(string[] args)
+#pragma warning restore UseAsyncSuffix // Use Async suffix
+        {
+            var dict = new Dictionary<string, string>
+            {
+                ["allowestsrnonmsi"] = "true"
+            };
+
+            string secret = "Lab_Secret"; //Lab secret from TestConstants.MsalCCAKeyVaultUri
+
+            //Uncomment below line to test in dev env.
+            //Environment.SetEnvironmentVariable("REGION_NAME", "centralus");
+            string[] scopes = new string[] { $"{clientId}/.default", };
+            var cca = ConfidentialClientApplicationBuilder.Create(clientId)
+                .WithAuthority("https://login.windows.net/72f988bf-86f1-41af-91ab-2d7cd011db47")
```

Comment:
>login.windows.net/ [](start = 40, length = 18)

any reason the old windows.net is used instead of onmicrosoftonline.com?

Reply:
No, I can change it to onmicrosoftonline.com

Reply:
i still see login.windows.net, i think this is still open.

---
In reply to: [508175872](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2119#discussion_r508175872) [](ancestors = 508175872)

Reply:
`login.windows.net` is a valid alias ... also testing that aliasing still works :)